src/transformers/pipelines/zero_shot_classification.py

```diff
@@ -100,13 +132,17 @@ def _parse_and_tokenize(
             truncation=truncation,
         )
 
+        if self.model.config.model_type == "t5":
+            inputs["decoder_input_ids"] = decoder_inputs.input_ids
+
         return inputs
 
     def __call__(
         self,
         sequences: Union[str, List[str]],
         candidate_labels,
         hypothesis_template="This example is {}.",
+        prefix="mnli hypothesis: {hypothesis} premise: {premise}",
```

Comment:
This is probably overkill to ADD a variable.

Fusing `prefix` and `hypothesis_template` into something lke `template` might be interesting (also wouldn't we want to be able to choose the decoder inputs?

Reply:
I have kept them separated to make them compatible with the original zero-shot pipeline. The `prefix` is only used when the model is a T5 like model. A prompt for the decoder inputs should be fine as well.

Fusing both together will make a model dependant template, which in principle I understand that is not desired. What do you think?